### PR TITLE
Add user stories to the DG

### PR DESCRIPTION
**Pull request by tiongMax**
Highlighted user stories have the highest priorities (3 *s). The rest are currently having 2 *s.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -282,14 +282,40 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* * *`  | volunteer coordinator  | create new events by specifying the date and time of the event                        | check the date and time of an event in the future               |
+| `* * *`  | volunteer coordinator  | change the name of the event                                                          | update an event with the correct name                           |
+| `* * *`  | volunteer coordinator  | change the date and time of an event                                                  | update an event with the correct date and time                  |
+| `* * *`  | volunteer coordinator  | check the list of events through the app                                              | view all upcoming events                                        |
+| `* * *`  | volunteer coordinator  | check the date and time of an event                                                   | know when and what time an event will occur                     |
+| `* * *`  | volunteer coordinator  | delete volunteer opportunities                                                        | remove them when they are no longer needed                      |
```
Comment (zacwong2151) [inline docs/DeveloperGuide.md:293]: I think `delete volunteer events` is clearer in showing that you want to delete **events**

**Review by zacwong2151 — COMMENTED** [review]
LGTM apart from one minor part

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:293]: yea agree, should be consistent in terminology

**Review by jordankanghm — APPROVED** [review]
Seems like all the points are here! Perhaps you could group related stories into supers(e.g. all create event related stories to be grouped into a super with heading "As a volunteer coordinator, I want to be able to create new events") to make everything more organised.

Besides that lgtm assuming it renders fine on the website!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (brein62) [reply docs/DeveloperGuide.md:293]: or rather, just `delete events` to confuse the users less

**Review by brein62 — CHANGES_REQUESTED** [review]
LGTM! only just one minor suggestion suggested by zac and jordan that i think its important to make clear

**Review by zacwong2151 — APPROVED** [review]

**Review by brein62 — APPROVED** [review]
LGTM!

**Pull request merged**